### PR TITLE
O3-1524: Implement form viewer component

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
### Purpose
- Implement the form viewer component to OpenMRS form builder using the OHRI form engine.
- This component allows users to see a preview of the building form.

### Screenshots
Screenshot 2022-09-14 at 11 36 44 PM

https://user-images.githubusercontent.com/27630091/190230022-35f01c9c-06e9-4da7-a4ad-bdfa65226aca.mov

### CODE REVIEW

File: src/components/form-editor/form-renderer/form-renderer.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const FormRenderer: React.FC = () => {
+  const defaultSchema: OHRIFormSchema = {
+    name: "",
+    pages: [],
+    processor: "EncounterFormProcessor",
+    uuid: "xxx",
+    encounterType: "",
+    referencedForms: [],
+  };
+  const patientUUID = "b280078a-c0ce-443b-9997-3c66c63ec2f8";
```
Comment (samuelmale) [inline src/components/form-editor/form-renderer/form-renderer.tsx:15]: This should be driven through a config, what happens if I ran this app against an OpenMRS instance without a patient with uuid: `"b280078a-c0ce-443b-9997-3c66c63ec2f8"`?

**Review by samuelmale — CHANGES_REQUESTED** [review]
Almost there, just one minor comment.

**Comment by kumuditha-udayanga** [comment]
Thank you for the review @samuelmale, the requested change is made

**Review by samuelmale — APPROVED** [review]
LGTM

**Pull request merged**